src/main/scala/dpla/ingestion3/model/package.scala

```diff
@@ -247,25 +247,14 @@ package object model {
         |       | local_id = ${record.sourceResource.identifier.map(escapeWikiChars).mkString("; ")}
         |   }}
         |   | Institution = {{ Institution | wikidata = $dataProviderWikiUri }}
-        |   | Other fields = ${getWikiOtherFieldsRights(record.edmRights)}
         | }}""".stripMargin
   }
 
-  def getWikiOtherFieldsRights(edmRights: Option[URI]): String =
-    edmRights match {
-      case Some(uri) =>
-        if(uri.toString.startsWith("http://rightsstatements.org"))
-          s"{{ InFi | Standardized rights statement | {{ rights statement | ${escapeWikiChars(uri.toString)} }} }}"
-        else
-          ""
-      case None => ""
-    }
-
-
   def getWikiPermissionTemplate(edmRights: Option[URI]): String = {
     edmRights match {
       case Some(uri) => uri.toString match {
-        case t if t.startsWith("http://rightsstatements.org/vocab/NoC-US/") => "PD-US"
+        case t if t.startsWith("http://rightsstatements.org/vocab/NKC/") => "NKC | $dataProviderWikiUri"
```

Comment:
@DominicBM this shouldn't be adding the $dataProviderWiki URI here because it doesn't have access to that value in the method.

Reply:
It should be added in the markup function `buildWikiMarkup`